experimental/traffic-portal/nightwatch/dataClient.ts

```diff
@@ -27,7 +27,7 @@ import {
 	RequestCacheGroup,
 	RequestCoordinate,
 	RequestDeliveryService,
-	RequestDivision,
+	RequestDivision, RequestParameter,
```

Comment:
please put this on its own line, in keeping with the rest of the import statement